LICENSE

```diff
@@ -1,21 +1,661 @@
-MIT License
-
-Copyright (c) 2020-2022 Jeroen Willemsen and WrongSecret contributors.
-
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
-
-The above copyright notice and this permission notice shall be included in all
-copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
+                    GNU AFFERO GENERAL PUBLIC LICENSE
+                       Version 3, 19 November 2007
+
+ Copyright (C) 2007 Free Software Foundation, Inc. <https://fsf.org/>
```

Comment:
Can you please change this to a similar header as we have with MIT?

Reply:
No, this license file should not be modified. Added a line to the README.md instead

Reply:
ahhh thanks! I guess we need to do the same for https://owasp.org/www-project-wrongsecrets/#:~:text=secret%20detection%20testbed.-,Licensing,-This%20program%20is right ?

Reply:
Can you check https://github.com/OWASP/www-project-wrongsecrets/pull/8 please? so we can merge them both in :). and then maybe have a blog on why we changed licenses :) ?